sw.js

```diff
@@ -33,7 +33,7 @@ workbox.routing.registerRoute(
 
 workbox.routing.registerRoute(
   // Cache immutable files forever
-  /^https:\/\/en\.wikipedia\.org\//,
+  /^https:\/\/..\.wikipedia\.org\//,
```

Comment:
Good catch. There are sometimes 3 chars. See https://en.wikipedia.org/wiki/List_of_Wikipedias#Detailed_list

Reply:
made it more generic